lib/services/kramdown_service.rb

```diff
@@ -113,13 +114,14 @@ def create_jekyll_post_text(text, author, title, tags, overlay, hero)
       lead_break_section = "{: .lead}\r\n<!–-break-–>"
 
       hero_to_use = hero
+      hero_to_use = DEFAULT_HERO if hero_to_use.empty?
       result = %(---
 layout: post
 title: #{title}
 author: #{author}\r\n)
 
-      result << "#{tag_section}\r\n" unless parsed_tags.empty?
-      result << %(hero: #{hero_to_use}
+      result += "#{tag_section}\r\n" unless parsed_tags.empty?
```

Comment:
Just curious on why you changed this? The << and += operator do the same thing so was this causing issues?

Reply:
Shift operator mutates :(